common.blocks/i-bem/i-bem.vanilla.js

```diff
@@ -257,12 +257,15 @@ var BemEntity = inherit(events.Emitter, /** @lends BemEntity.prototype */ {
     /**
      * Checks whether a BEM entity has a modifier
      * @param {String} modName Modifier name
-     * @param {String} [modVal] Modifier value
+     * @param {String|Boolean} [modVal] Modifier value. If not of type String or Boolean, it is casted to String
      * @returns {Boolean}
      */
     hasMod : function(modName, modVal) {
+        var withModVal = arguments.length === 2;
```

Comment:
Can we decide that `hasMod('mod1', undefined)` is a useless edge case? At least until someone will catch this.